src/Utils/Hexagons.svelte

```diff
@@ -1,16 +1,23 @@
 <script>
-import { block0_31, block32_63, block64_95, block96_127, act_command } from "../../stores/stores.js"
-import Communication from "./Communication.svelte"
-import Moveable from "svelte-moveable"
+import { block0_31, block32_63, block64_95, block96_127, act_command, message, OP_Mode, preset_display } from "../../stores/stores.js";
+import Communication from "./Communication.svelte";
+import Moveable from "svelte-moveable";
+import { onMount } from "svelte";
 
 
 export let test_ui = false;
-export let activeHexagon = -1;
-export let message = 'starting ...';
-
+export let activeHexagon = [];
+export let orientation = "horizontal";
+export let arraySize = "normal";
```

Comment:
Mathematical arguments will make this much easier to use in other situations. I expect overlaying the hexagons on the smaller options will involve a refactor of these arguments or nesting of components that compute the arguments.